pkg/camo/proxy.go

```diff
@@ -221,10 +221,17 @@ func (p *Proxy) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 
 	switch resp.StatusCode {
 	case 200:
-		// check content type
+		contentType := resp.Header.Get("Content-Type")
+
+		if contentType == "" {
+			mlog.Debug("Missing content-type")
+			http.Error(w, "Missing content-type", http.StatusBadRequest)
```

Comment:
Can you change these two strings to `"Empty content-type returned"`?

Reply:
Fixed in 6a8c69a